accession-commons-monotonic-generator-jpa/src/test/java/uk/ac/ebi/ampt2d/commons/accession/core/BasicMonotonicAccessioningWithAlternateRangesTest.java

```diff
@@ -75,12 +77,12 @@ public void testRecoverState() {
         // create 3 un-complete contiguous id blocks of size 10
         // block-1 : (100 to 109), block-2 : (110 to 119), block-3 : (120 to 129)
         List<ContiguousIdBlock> uncompletedBlocks = new ArrayList<>();
-        uncompletedBlocks.add(new ContiguousIdBlock(categoryId, instanceId2, 100, 10));
-        uncompletedBlocks.add(new ContiguousIdBlock(categoryId, instanceId2, 110, 10));
-        uncompletedBlocks.add(new ContiguousIdBlock(categoryId, instanceId2, 120, 10));
+        uncompletedBlocks.add(getUnreservedContiguousIdBlock(categoryId, instanceId2, 100, 10));
+        uncompletedBlocks.add(getUnreservedContiguousIdBlock(categoryId, instanceId2, 110, 10));
+        uncompletedBlocks.add(getUnreservedContiguousIdBlock(categoryId, instanceId2, 120, 10));
         contiguousIdBlockService.save(uncompletedBlocks);
 
-        assertEquals(3, contiguousIdBlockService.getUncompletedBlocksByCategoryIdAndApplicationInstanceIdOrderByEndAsc(categoryId, instanceId2).size());
+        assertEquals(3, getUncompletedBlocksByCategoryIdAndApplicationInstanceIdOrderByEndAsc(contiguousIdBlockService.getRepository(), categoryId, instanceId2).size());
```

Comment:
The helper method is being used only in asserting the DB state for uncompleted blocks and not as a replacement for any actual method of the service.

Earlier, the actual service method's functionality was such that it could be used here for assertions of uncompleted blocks but due to the code update, that is no longer possible.

Reply:
Sorry was misreading these tests - makes sense!